torch/_inductor/async_compile.py

```diff
@@ -63,8 +63,8 @@ def pre_fork_setup():
         from triton.compiler.compiler import triton_key
 
         triton_key()
-    except ModuleNotFoundError:
-        # Might not be installed.
+    except ImportError:
```

Comment:
This seems to capture both of triton does not exist and if triton.compiler.compiler.triton_key does not exist